exasol_sphinx_github_pages_generator/deployer.py

```diff
@@ -61,7 +61,8 @@ def check_out_source_branch_as_worktree(self,
                      self.source_branch, "--force"], check=True)
                 print(f"Successfully added new temp worktree for source branch {self.source_branch}, "
                       f"and checked out (Local or stashed changes will be ignored in this build).")
-                os.chdir(f"{self.worktree_paths['source_worktree']}/doc")
+                # change into documentation source dir
+                os.chdir(f"{self.worktree_paths['source_worktree']}{self.source_dir}")
```

Comment:
it seems that source_dir is '/doc/' string. You get it as 'doc' and you can use os.path.join rather than string operation to create path.

Reply:
yes, this is also a problem in other parts of this project. i suggest i make a new issue to replace them all, and ignore it here for now.